scratch/evaluate_scheduler_qd_dense_scenario.cc

```diff
@@ -0,0 +1,825 @@
+  /** TCP Variants **/
+  tcpVariants.insert (make_pair ("NewReno",       "ns3::TcpNewReno"));
+  tcpVariants.insert (make_pair ("Hybla",         "ns3::TcpHybla"));
+  tcpVariants.insert (make_pair ("HighSpeed",     "ns3::TcpHighSpeed"));
+  tcpVariants.insert (make_pair ("Vegas",         "ns3::TcpVegas"));
+  tcpVariants.insert (make_pair ("Scalable",      "ns3::TcpScalable"));
+  tcpVariants.insert (make_pair ("Veno",          "ns3::TcpVeno"));
+  tcpVariants.insert (make_pair ("Bic",           "ns3::TcpBic"));
+  tcpVariants.insert (make_pair ("Westwood",      "ns3::TcpWestwood"));
+  tcpVariants.insert (make_pair ("WestwoodPlus",  "ns3::TcpWestwoodPlus"));
+
+  /* Command line argument parser setup. */
+  CommandLine cmd;
+  // cmd.AddValue ("applicationType", "Type of the Tx Application: onoff or bulk", applicationType);
+  // cmd.AddValue ("packetSize", "Application packet size [bytes]", packetSize);
+  cmd.AddValue ("dataRate", "Application data rate", appDataRate);
+  // cmd.AddValue ("maxPackets", "Maximum number of packets to send", maxPackets);
+  // cmd.AddValue ("tcpVariant", "Transport protocol to use: TcpHighSpeed, TcpVegas, TcpNewReno, TcpWestwood, TcpWestwoodPlus", tcpVariant);
+  // cmd.AddValue ("socketType", "Socket type (default: ns3::UdpSocketFactory)", socketType);
+  // cmd.AddValue ("bufferSize", "TCP Buffer Size (Send/Receive) [bytes]", bufferSize);
+  // cmd.AddValue ("msduAggregation", "The maximum aggregation size for A-MSDU [bytes]", msduAggregationSize);
+  cmd.AddValue ("mpduAggregation", "The maximum aggregation size for A-MPDU [bytes]", mpduAggregationSize);
+  // cmd.AddValue ("queueSize", "The maximum size of the Wifi MAC Queue [packets]", queueSize);
+  // cmd.AddValue ("frameCapture", "Use a frame capture model", frameCapture);
+  // cmd.AddValue ("frameCaptureMargin", "Frame capture model margin [dB]", frameCaptureMargin);
+  cmd.AddValue ("phyMode", "802.11ad PHY Mode", phyMode);
+  // cmd.AddValue ("verbose", "turn on all WifiNetDevice log components", verbose);
+  cmd.AddValue ("simulationTime", "Simulation time [s]", simulationTime);
+  // cmd.AddValue ("qdChannelFolder", "The name of the folder containing the QD-Channel files", qdChannelFolder);
+  // cmd.AddValue ("numSTAs", "The number of DMG STA", numSTAs);
+  // cmd.AddValue ("pcap", "Enable PCAP Tracing", pcapTracing);
+  cmd.AddValue ("scheduler", "The type of scheduler to use in the simulation", schedulerType);
+  cmd.AddValue ("interAllocation", "Duration of a broadcast CBAP between two ADDTS allocations [us]", interAllocDistance);
+  // cmd.AddValue ("csv", "Enable CSV output instead of plain text. This mode will suppress all the messages related statistics and events.", csv);
+  cmd.AddValue ("logComponentsStr", "Components to be logged from tLogStart to tLogEnd separated by ':'", logComponentsStr);
+  // cmd.AddValue ("tLogStart", "Log start [s]", tLogStart);
+  // cmd.AddValue ("tLogEnd", "Log end [s]", tLogEnd);
+  cmd.Parse (argc, argv);
```

Comment:
What about cleaning up all the unused cmd? Are you keeping them "just in case"?

Reply:
Yes!
I commented most of them for compatibility with SEM. We can restore them maybe